ansible/roles/druid-ingestion/templates/raw_telemetry_events

```diff
@@ -60,6 +60,11 @@
               "name": "context_env",
               "expr": "$.context.env"
             },
+            {
+              "type": "path",
```

Comment:
`context.hub` field is not present in the generic telemetry. This is a specific field for iGot. The sunbird telemetry ingestion will fail if we merge this change into the raw_telemetry_events ingestion spec.